eacl/table.go

```diff
@@ -19,6 +19,31 @@ type Table struct {
 	records []Record
 }
 
+// CopyTo writes deep copy of the [Table] to dst.
+func (t Table) CopyTo(dst *Table) {
+	if dst == nil {
+		return
+	}
+
+	t.version.CopyTo(&dst.version)
+
+	dst.cid = nil
+	if t.cid != nil {
+		var id cid.ID
+		t.cid.CopyTo(&id)
+
+		dst.cid = &id
+	}
+
+	dst.records = make([]Record, 0, len(t.records))
```

Comment:
same as  in `Record`

Reply:
`t.records[i].CopyTo(&dst.records[i])` works fine

Reply:
Updated